docs/concepts/did.md

```diff
@@ -2,3 +2,103 @@
+Note that the identifier of the two DIDs, apart from the initial `01` sequence of the light DID, are equal since both DIDs are derived from the same KILT account.
+
+Once a light DID is migrated, all the credentials collected by the light DID can only be presented using the migrated on-chain DID.
+This is by design, as it is assumed that the user had valid reasons to migrate the DID on the chain, and as on-chain DIDs offer greater security guarantees, KILT will reject light DID signatures even in case the original claim in the attestation was generated with that off-chain DID.
+
+## DIDs and Web3 names
+
+Web3 names are user-friendly aliases for KILT DIDs.
+They serve the same purpose that domain names do for IP addresses: who knows the IP address under the `kilt.io` domain name? 🤷🏽‍♀️
+There is a one-to-one relationship between DIDs and Web3 names.
+This means that a KILT DID can be linked to at most one Web3 name, and a Web3 name can be claimed only by one DID.
+
+Each Web3 name is globally unique within the KILT space, and is composed of a sequence of a minimum of 3 to a maximum of 32 characters taken from a specific character set to enhance human redability and reduce the chances of "two Web3 names looking the same, despite being different".
+The character set include only:
+- *lowercase letters*, from `a` to `z`
+- *digits* from `0` to `9`
+- the symbols `-` and `_`
+
+A regex that would match all and only the allowed Web3 names would be the following:
+
+```
+^[a-z0-9\_\-]{3,32}$
+```
+
+In the global URI space, Web3 names are prefixed with the `w3n:` URI namespace. 
+For example, the full URI for the Web3 name `example-web3name` is `w3n:example-web3name`.
+
+### Linking accounts to a Web3 name
+
+Beyond linking a Web3 name, KILT allows DID owners to link multiple accounts to a single DID.
+These accounts are not specific to the KILT blockchain but can reference any chain within the Dotsama ecosystem.
+Each account <-> DID link requires the payment of a small deposit, which is returned whenever the link is removed.
+
+For DIDs that have also claimed a Web3 name, the linking feature opens up the way to a host of possibilities, e.g., showing the Web3 name of a collator's account on the [KILT stakeboard](https://stakeboard.kilt.io/).
+
+<ThemedImage
+  alt="DID lookup diagram"
+  sources={{
+    light: '/img/concepts/did/did-lookup-light.png',
+    dark: '/img/concepts/did/did-lookup-dark.png',
+  }}
+/>
+
+:::caution
+While multiple accounts can be linked to a DID, it is important to notice the difference between the two.
+
+KILT *accounts* are classical blockchain accounts, that can be used to hold and send KILT tokens, as well as signing and submitting transactions.
+On the other hand, KILT *DIDs* are a L2 construct which are derived from KILT accounts, but are completely separated from them.
```

Comment:
I'm not sure if L2 is understood widely.

```suggestion
On the other hand, KILT *DIDs* are a higher level construct which are derived from KILT accounts, but are completely separated from them.
```

Reply:
https://github.com/KILTprotocol/docs/pull/82/commits/68ce5d9232ea37164d25c622112f8ae659f344b7